source/dotnet/DomainTests/DomainTests.csproj

```
@@ -7,22 +7,22 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Energinet.DataHub.Core.FunctionApp.TestCommon" Version="3.5.0" />
-    <PackageReference Include="Energinet.DataHub.Wholesale.Client" Version="3.4.0" />
+    <PackageReference Include="Energinet.DataHub.Core.FunctionApp.TestCommon" Version="4.0.1" />
+    <PackageReference Include="Energinet.DataHub.Wholesale.Client" Version="5.0.24" />
```

bump this to version 6.0.0 and i can close this https://github.com/Energinet-DataHub/opengeh-wholesale/pull/945
```suggestion
    <PackageReference Include="Energinet.DataHub.Wholesale.Client" Version="6.0.0" />
```